main/entry.c

```diff
@@ -1361,7 +1423,9 @@ static void writeTagEntry (const tagEntryInfo *const tag)
 		return;
 
 	DebugStatement ( debugEntry (tag); )
-	if (Option.xref)
+	if (Option.json)
```

Comment:
(masatake should do): main part should provide pluggin interface for replacing output handler